mmv1/api/resource/examples.go

```diff
@@ -162,11 +168,32 @@ func (e *Examples) UnmarshalYAML(n *yaml.Node) error {
 		return err
 	}
 
-	e.ConfigPath = fmt.Sprintf("templates/terraform/examples/%s.tf.erb", e.Name)
+	e.ConfigPath = fmt.Sprintf("templates/terraform/examples/go/%s.tf.tmpl", e.Name)
+	e.SetHCLText()
 
 	return nil
 }
 
+func (e *Examples) SetHCLText() {
+	templatePath := e.ConfigPath
+	templates := []string{
+		templatePath,
+	}
+	templateFileName := filepath.Base(templatePath)
+
+	tmpl, err := template.New(templateFileName).ParseFiles(templates...)
+	if err != nil {
+		glog.Exit(err)
+	}
+
+	contents := bytes.Buffer{}
+	if err = tmpl.ExecuteTemplate(&contents, templateFileName, e); err != nil {
+		glog.Exit(err)
+	}
```

Comment:
We can make the template execution a separate function later if it is used by multiple places in the future.

Reply:
yes I was thinking this, but wasn't sure exactly how others would play out yet.